packages/integration-tests/src/fixtures/storage/storage-hosts.ts

```diff
@@ -285,6 +286,7 @@ const hosts = [
     registered_at: '2022-08-16T13:26:48.075Z',
     requested_hostname: 'storage-test-odf-worker-1',
     role: 'worker',
+    skip_formatting_disks: '/dev/disk/by-path/pci-0000:00:1f.2-ata-40',
```

Comment:
I think it would be great to update the fixture to have 2 formatteable disks.
Then we could verify that the format column is checked for one disk and not for the other.